### PR TITLE
Add source system name to digital specimen and digital media

### PR DESCRIPTION
**Pull request by samleeflang**
- Add source system name to digital specimen and digital media 
- Added sourceSystemName to schema's 
- Created new component to hold the information. This component can be injected where the info is needed. This way we don't need to pass it all the way down the line to the director, but we can just inject it there.

### CODE REVIEW

File: src/main/java/eu/dissco/core/translator/component/SourceSystemComponent.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package eu.dissco.core.translator.component;
+
+import eu.dissco.core.translator.properties.WebClientProperties;
+import eu.dissco.core.translator.repository.SourceSystemRepository;
+import lombok.Getter;
+import org.springframework.stereotype.Component;
+
+@Getter
```
Comment (southeo) [inline src/main/java/eu/dissco/core/translator/component/SourceSystemComponent.java:8]: I think you just want a getter on source system name/id/endpoint, not on repository or webClientProperties

**Review by southeo — COMMENTED** [review]
:soap:

File: src/main/java/eu/dissco/core/translator/component/SourceSystemComponent.java (hunk 1 above)
Comment (southeo) [reply src/main/java/eu/dissco/core/translator/component/SourceSystemComponent.java:8]: comment still stands

Comment (samleeflang) [reply src/main/java/eu/dissco/core/translator/component/SourceSystemComponent.java:8]: They are no longer exposed as internal attributes so didn't get a getter method

**Review by southeo — APPROVED** [review]
:bubbles:

**Pull request merged**